references/validationFunctions/sample_registration/specimentypeDesignation.js

```diff
@@ -1,5 +1,8 @@
-const validation = () => {
-    function validate() {
+/**
+ * Checks that tumour desigation can only be normal iFF the specimen types are consered normal
+ */
+const validation = ($row, $field) => 
```

Comment:
does this work with clinical ?

Reply:
I mean injecting $row and $field this way

Reply:
yes, because this part of the "outer" function gets removed by `/references/script.js` .

The script is converted in a string and only the inner part of the function gets added into the dictionary.